### ISSUE TITLE
Mango leaves need to last longer, they only last 1 minute and it's the only way to kill cows

### ISSUE THREAD

**Issue by DiscardedSlinky** [jasonrohrer/OneLifeData7#1216]
Basically just the title. You'd need a race car to try to kill cows in other biomes

**Comment by Rukhmar** [comment]
Yeah, cows are annoying since they can be spammed and don't die. Just like how dogs used to be before they were changed to get old and die. Please give us a way to kill them besides the mango leaf, or make the leaf last longer.
